seedDB/README.md

```diff
@@ -4,7 +4,32 @@ These files are MySQL database dump files for seeding a new instance of the cBio
 
 The database schema and cBioPortal release follows different numbering cycles since cBioPortal 1.5.0 and database schema 2.1.0. This means that the version numbers won't be identical. cBioPortal 1.9.0 with database schema 2.4.0 removed PDB annotations from the database.
 
-## Latest seed database
+Instructions for building and updating seedDBs [HERE](#for-developers)
+
+## Release Notes
+### Latest seed database
+
+#### Seed database schema 2.12.8
+
+This schema is required for cBioPortal release versions:
+- **2.0.0**
+
+**** This part needs revise****
```

Comment:
This refers to the cBioPortal version where the schema 2.12.8 was introduced. In this case is version 3.6.0. Therefore, this schema can be used for cBioPortal version 3.6.0 or higher (> 3.6.0).

Reply:
@oplantalech Thanks! updated the version here